### ISSUE TITLE
Vietnamese Translation version.

### ISSUE THREAD

**Issue by duythanhvn** [progit/progit2#1380]
Hello all,

I'm a developer in Vietnam. I would like to contribute a Vietnamese translation to this book. I looked through the two issues created earlier (#629, #988), it seems that no translation has been completed yet. 

Therefore, I would like to make a new copy at this repository: https://github.com/aiivn/progit2

It may take a while for me to finish, I will try to update the progress as instructed from the TRANSLATING.md file as soon as possible.

Kindly regards,
Thanh Doan

**Comment by jnavila** [comment]
Please contact us when you have translated a little. Then you can create an organization (named for instance `progit2-vi`) so that we can list your work in progit's TRANSLATION page.

**Comment by duythanhvn** [comment]
Hi @jnavila,
Now that this repository is in an organization, what you want is that I need to create a separate organization called progit2 with the suffix and only one repository for my language?

**Comment by jnavila** [comment]
Yes, that was my idea. But AFAIK, there's already a translation here: https://github.com/progit2-vi/progit2 even if it seems dead by now.

So, please start to translate the book in your repository, and when the first chapter is translated please get in contact with me, so that we can work on getting your work published on git-scm.com and listed in TRANSLATION.

**Comment by KZiemian** [comment]
What is the status of this issue?
